src/ovp/player-defaults.js

```diff
@@ -15,4 +23,5 @@ export function setDefaultAnalyticsPlugin(playerConfig: PKPlayerOptionsObject):
       }
     });
   }
+  Object.assign(playerConfig.plugins.kanalytics, {hasKanalony: true});
```

Comment:
need to make sure that kava is actually enabled now that we have ability to disable plugin